RealEstate/apps/core/templates/core/categories.html

```diff
@@ -124,7 +84,7 @@ <h2 class="modal-title">Edit your category!</h2>
         <form action="" role="form" method="post">
         	{% bootstrap_form editForm%}
         	{% csrf_token %}
-        	<div style="margin-top: 40px">
+        	<div id="saveButtonMargin">
```

Comment:
In this case it might be more tricky to style the button itself because it interacts with a form.